drivers/gpu/drm/bridge/cdns-mhdp.c

```diff
@@ -1382,6 +1382,7 @@ static int mhdp_link_training(struct cdns_mhdp_device *mhdp,
 	cdns_mhdp_reg_read(mhdp, CDNS_DP_FRAMER_GLOBAL_CONFIG, &reg32);
 	reg32 &= ~GENMASK(1, 0);
 	reg32 |= CDNS_DP_NUM_LANES(mhdp->link.num_lanes);
+	reg32 |= CDNS_DP_WR_FAILING_EDGE_VSYNC;
```

Comment:
Where is that recommendation? The docs I have say: "When set to 1 change the write state machine to sync on falling edge of vsync. Used only for debug purpose.". And reset value is 0.

Reply:
This is as per internal communication with HW people. It is also recommended to do case by case review while changing/removing registers/register bits which are unused/used for debug purpose.